regolith/schemas.py

```diff
@@ -1652,6 +1672,139 @@
             "type": "integer",
         },
     },
+    "refereeReports": {
+        "_description": {
+            "description": "This is a collection of information that will be "
+            "be used to build a referee report. This should probably be private."
+        },
+        "_id": {
+            "description": "short representation, such as date and journal",
+            "required": True,
+            "type": "string",
+        },
+        "claims": {
```

Comment:
Why do we need the `claims` level, should we have the `foundWhat` and `whyImportant` at the main level? I think this will solve the issue with the CI. I think the schema is not validating because it thinks claims is a dict, with a dict inside it? (maybe not exactly certain). For an example of dict schemas you might look at the [teams](https://github.com/regro/regolith/blob/master/regolith/schemas.py#L966) chunk, note that it is a list of dicts.